src/panels/config/integrations/integration-panels/ozw/ozw-config-network.ts

```diff
@@ -0,0 +1,304 @@
+import "@material/mwc-fab";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import { navigate } from "../../../../../common/navigate";
+import "../../../../../components/ha-card";
+import "../../../../../components/ha-icon-next";
+import "../../../../../components/buttons/ha-call-service-button";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant, Route } from "../../../../../types";
+import "../../../ha-config-section";
+import { mdiNetwork, mdiCircle, mdiCheckCircle, mdiCloseCircle } from "@mdi/js";
+import "../../../../../layouts/hass-tabs-subpage";
+import type { PageNavigation } from "../../../../../layouts/hass-tabs-subpage";
+import "@material/mwc-button/mwc-button";
+import {
+  OZWInstance,
+  fetchOZWNetworkStatus,
+  fetchOZWNetworkStatistics,
+  networkOnlineStatuses,
+  networkOfflineStatuses,
+  networkStartingStatuses,
+  OZWNetworkStatistics,
+} from "../../../../../data/ozw";
+
+export const ozwTabs: PageNavigation[] = [
+  {
+    translationKey: "ui.panel.config.ozw.navigation.select_instance",
```

Comment:
Should we show this when there is only 1 instance?

Reply:
Good point. I pushed a commit to clear the list of tabs for the moment since we only have a single page. Will re-add the tabs in a future PR when we add additional pages (like the nodes list) to the panel